### DISCUSSION TITLE
QImg  - add errorSrc prop (like placeholderSrc)

### DISCUSSION THREAD

**Discussion by ShlomoCode** [quasarframework/quasar#16964] [Ideas / Proposals] [answered]
You can define an image that will appear when the image is **loaded** using a prop named [`placeholderSrc`](https://quasar.dev/vue-components/img#qimg-api)
But if you want to set an image that will appear when there is an **error** loading the image, you can only do this by using a slot and repeating all the `QImg` tag and its properties, to get a consistent look

I suggest adding an `errorSrc` prop where it will be possible to pass a path to the image that will appear when there is an error loading the main image

**Comment by ShlomoCode** [comment]
@rstoenescu Are you interested in me trying to make a PR of this?

**Reply by rstoenescu** [reply]
Yes, please do! Thanks!

**Reply by ShlomoCode** [reply]
@rstoenescu What should be the behavior in the case that both prop `errorSrc` and `error` slot are defined?

**Reply by rstoenescu** [reply]
The slots should always win. When the slot is also specified the errorSrc becomes redundant.
Created an issue from this thread. Let's continue discussion there: https://github.com/quasarframework/quasar/issues/16969

**Reply by ShlomoCode** [reply]
@rstoenescu looks like I won't submit a PR for this, it's bigger than I thought, sorry...

**Reply by rstoenescu** [reply]
What seems to be the issue?

**Reply by ShlomoCode** [reply]
Not a particular difficulty, I just saw that it's a bigger change than I thought, and I can't find the time to finish ([I started](https://github.com/ShlomoCode/quasar/commit/83e41f88ccf5935ee0c06ebfe0c8448cede33c56))

**Reply by rstoenescu** [reply]
Implemented in https://github.com/quasarframework/quasar/commit/b0e2ef8dfbac9331a4fee685bd68dc9395f17a14
Available in Quasar v2.15.0 already.

Thanks for all your efforts and for wanting to contribute!

**Comment by rstoenescu** [comment, marked as answer]
Implemented in https://github.com/quasarframework/quasar/commit/b0e2ef8dfbac9331a4fee685bd68dc9395f17a14

Will be available in Quasar v2.15.0
